Makefile

```diff
@@ -14,21 +13,25 @@ help: ## Print this help message
 
 .PHONY: create-switch
 create-switch: ## Create opam switch
-	opam switch create . -y --deps-only
+	opam switch create . 4.14.2 -y --deps-only
```

Comment:
I'm not aware of a 4.14.2 version of OCaml.

Reply:
I just followed what opam was telling me:

```
$ opam list ocaml | grep 4.14.2
ocaml.4.14.2 4.14.0      The OCaml compiler (virtual package)
```

Reply:
Not sure why that's the case, but let's stick to 4.14.1, which is a released version.